nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js

```diff
@@ -67,7 +67,10 @@ export function GatewayJsonConfig() {
 
   return (
     <JsonEditor
-      content={gwInfoJson}
+      content={{
+        ...gwInfoJson,
+        connected_enodeb_serials: gwInfoJson.connected_enodeb_serials ?? [],
```

Comment:
Can you make a small change to GatewayContext setState to accept the flow parameter of mutable_lte_gateway rather than lte_gateway. 

export type GatewayContextType = {
  state: {[string]: lte_gateway},
-  setState: (key: gateway_id, val?: lte_gateway) => Promise<void>,
-  setState: (key: gateway_id, val?: mutable_lte_gateway) => Promise<void>,
  updateGateway: (props: $Shape<UpdateGatewayProps>) => Promise<void>,
};

We should ensure that we only pass in the mutable_lte_gateway for saving in the config. Also let us do a get to store the latest state in gatewayState For e.g.

export async function setSubscriberState(props: SubscriberStateProps) {
  const {networkId, subscriberMap, setSubscriberMap, key, value} = props;
  if (value != null) {
    if (key in subscriberMap) {
      await MagmaV1API.putLteByNetworkIdSubscribersBySubscriberId({
        networkId,
        subscriber: value,
        subscriberId: key,
      });
    } else {
      await MagmaV1API.postLteByNetworkIdSubscribers({
        networkId,
        subscriber: value,
      });
    }
    const subscriber = await MagmaV1API.getLteByNetworkIdSubscribersBySubscriberId( <=== we should do the same in gateway state as well. 
      {
        networkId,
        subscriberId: key,
      },
    );
    if (subscriber) {
      const newSubscriberMap = {...subscriberMap, [key]: subscriber};
      setSubscriberMap(newSubscriberMap);
    }
  } else {
    await MagmaV1API.deleteLteByNetworkIdSubscribersBySubscriberId({
      networkId,
      subscriberId: key,
    });
    const newSubscriberMap = {...subscriberMap};
    delete newSubscriberMap[key];
    setSubscriberMap(newSubscriberMap);
  }
}